doc/releases/mimic.rst

```diff
@@ -388,6 +388,11 @@ These changes occurred between the Luminous and Mimic releases.
     via a "pass through" method. See http://docs.ceph.com/docs/master/mgr/restful
     for details.
 
+  - New command to track throughput and IOPS statistics, also available in
+    `ceph -s` and previously in `ceph -w`. To use this command, enable
```

Comment:
all of these ceph commands need double backticks so that they are properly formatted as monospaced in HTML